src/modules/esl-event-listener/core/targets/resize.adapter.ts

```diff
@@ -58,17 +59,27 @@ export class ESLResizeObserverTarget extends SyntheticEventTarget {
 
     super.addEventListener('resize', callback);
     if (this.getEventListeners('resize').length > 1) return;
-    ESLResizeObserverTarget.observer$$.observe(this.target);
+
+    if (this.target instanceof Window) {
+      ESLEventUtils.subscribe(this, {event: 'resize', target: window}, ESLResizeObserverTarget.handleChange);
```

Comment:
I'd still prefer internal API independent of our API changes regarding the module facade. 
```
ESLEventListeners.subscribe(this, {event: 'resize', target: window}, ESLResizeObserverTarget.handleChange);
```